README.md

```diff
@@ -137,7 +137,7 @@ on the sniffs, the following installation steps are required.
 
 #### Issues when running the PHPCS Unit tests for another standard
 
-This sniff library uses it's own PHPUnit setup rather than the PHPCS native unit testing framework to allow for testing the sniffs with various config settings for the `testVersion` variable.
+This sniff library uses its own PHPUnit setup rather than the PHPCS native unit testing framework to allow for testing the sniffs with various config settings for the `testVersion` variable.
```

Comment:
Darn.. my mistake, it *was* a typo - I misread the added/removed lines. Disregard me. I'm tried & still with my head in a forest.